packages/react/src/components/Accordion/AccordionItem.js

```diff
@@ -16,15 +16,15 @@ import { useId } from '../../internal/useId';
 import deprecate from '../../prop-types/deprecate';
 
 const { prefix } = settings;
-const defaultRenderExpando = (props) => <button type="button" {...props} />;
+const defaultRenderExpand = (props) => <button type="button" {...props} />;
 
 function AccordionItem({
   children,
   className: customClassName,
   iconDescription, // eslint-disable-line
   open = false,
   onHeadingClick,
-  renderExpando: Expando = defaultRenderExpando,
+  renderExpand: Expand = defaultRenderExpand,
```

Comment:
One thing that I don't know if we decided on from the expando -> expand prop change was when we want to use expand versus toggle.

Does it make sense to refer to this as a toggle in the form as `renderToggle`? Expand implies on where-as toggle implies on/off, if I remember the terminology that we were using.